tests/annotations/tests.py

```diff
@@ -1167,6 +1170,28 @@ def test_alias_forbidden_chars(self):
                 with self.assertRaisesMessage(ValueError, msg):
                     Book.objects.annotate(**{crafted_alias: Value(1)})
 
+    @skipUnless(connection.vendor == "postgresql", "PostgreSQL tests")
+    @skipUnlessDBFeature("supports_json_field")
+    def test_set_returning_functions(self):
+        """Test that utilizing the set returning flag returns the expected count"""
+        from django.db.models import JSONField
+
```

Comment:
Move to the top, it can always be imported even when the backend doesn't support using it.